CONTRIBUTING.md

```diff
@@ -53,6 +53,71 @@ A team member will review the pull request as soon as possible.
 
 Please assign yourself to a maximum of 1 issue at a time.
 
+## Adding a new game
+
+<details>
+<summary>To add a new game, you need to follow these steps (<em>click to expand</em>)</summary>
+
+1. Create a new subfolder for your game within the `src` folder. In this example we will call it `chess`
+2. Create an `index.html` file containing this template code:
+
+   ```html
+   <!DOCTYPE html>
+   <html lang="en">
+     <head>
+       <meta charset="UTF-8" />
+       <title>chess</title>
```

Comment:
Consider adding `<meta name="description" content="A collection of for-fun mini-games" />` as all other mini-games have a description. I think it's still ok to leave it out if this is supposed to be a super basic guide to getting things set up.